cmd/servicelog/list.go

```diff
@@ -19,8 +19,12 @@ var listCmd = &cobra.Command{
 	Use:   "list",
 	Short: "gets all servicelog messages for a given cluster",
 	// validate only clusterid is provided
-	Args: cobra.ExactArgs(1),
+	Args: cobra.ArbitraryArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			cmd.Help()
+			return fmt.Errorf("the argument count is not equal to 1, it is %d", len(args))
```

Comment:
Can we change this error message to something a bit more descriptive?

Something like `cluster-identifier was not provided. Please provide a cluster ID, UUID, or name for this command`?

The difference is how the output will read:

```
$ ./osdctl servicelog list
gets all servicelog messages for a given cluster

Options:
  -A, --all-messages=false: Toggle if we should see all of the messages or only SRE-P specific ones

Usage:
  osdctl servicelog list [flags] [options] cluster-id

Use "osdctl options" for a list of global command-line options (applies to all commands).
Error: the argument count is not equal to 1, it is 0
the argument count is not equal to 1, it is 0
```

vs

```
$ ./osdctl servicelog list
gets all servicelog messages for a given cluster

Options:
  -A, --all-messages=false: Toggle if we should see all of the messages or only SRE-P specific ones

Usage:
  osdctl servicelog list [flags] [options] cluster-id

Use "osdctl options" for a list of global command-line options (applies to all commands).
Error: No Cluster Identifier provided.
No Cluster Identifier provided.
```